scss/boost_union/post.scss

```diff
@@ -1185,68 +1190,195 @@ body.theme-boost-union-commincourse {
             width: calc(98% - #{$blockregionoutsiderightwidth} - #{$blockregionoutsideleftwidth});
         }
 
-        /* Set the width and margin of the outside-left region. */
-        #theme-block-region-outside-left {
-            margin-right: 1rem;
-            margin-top: 1.5rem;
-            width: $blockregionoutsideleftwidth;
+        /* Set the width of the outside-* regions to auto.
+           Using a 100% width in the main inner area would push the side regions outside the page. */
+        #page .main-inner-outside-left-right .main-inner,
+        #page .main-inner-outside-left .main-inner,
+        #page .main-inner-outside-right .main-inner {
+            width: auto;
         }
+    }
+}
 
-        /* Set the width of the outside-right region. */
-        #theme-block-region-outside-right {
-            margin-left: 1rem;
-            margin-top: 1.5rem;
-            width: $blockregionoutsiderightwidth;
+/* And on larger screens and below. */
+@include media-breakpoint-down(md) {
+    /* Center the main region content while maintaining full width for the left and right block regions,
+    thereby positioning the page content in the middle of the page. */
+    body.limitedwidth {
+        .left-region-nextmaincontent {
```

Comment:
I cannot see that `left-region-nextmaincontent` class anywhere in Boost Union, so I assume this can safely be removed.